geo-specific-codelist/nuts-geography.cypher

```diff
@@ -0,0 +1,475 @@
+MERGE (node:`_code`:`_code_geography` { value:"UKD" });
+MATCH (parent:`_code_list`:`_code_list_nuts-geography`),(node:`_code`:`_code_geography` { value:"UKD" }) MERGE (node)-[:usedBy { label:"North West (England)"}]->(parent);
+MERGE (node:`_code`:`_code_geography` { value:"UKE" });
+MATCH (parent:`_code_list`:`_code_list_nuts-geography`),(node:`_code`:`_code_geography` { value:"UKE" }) MERGE (node)-[:usedBy { label:"Yorkshire and The Humber"}]->(parent);
+MERGE (node:`_code`:`_code_geography` { value:"UKF" });
+MATCH (parent:`_code_list`:`_code_list_nuts-geography`),(node:`_code`:`_code_geography` { value:"UKF" }) MERGE (node)-[:usedBy { label:"East Midlands (England)"}]->(parent);
+MERGE (node:`_code`:`_code_geography` { value:"UKG" });
+MATCH (parent:`_code_list`:`_code_list_nuts-geography`),(node:`_code`:`_code_geography` { value:"UKG" }) MERGE (node)-[:usedBy { label:"West Midlands (England)"}]->(parent);
+MERGE (node:`_code`:`_code_geography` { value:"UKH" });
+MATCH (parent:`_code_list`:`_code_list_nuts-geography`),(node:`_code`:`_code_geography` { value:"UKH" }) MERGE (node)-[:usedBy { label:"East of England"}]->(parent);
+MERGE (node:`_code`:`_code_geography` { value:"UKI" });
+MATCH (parent:`_code_list`:`_code_list_nuts-geography`),(node:`_code`:`_code_geography` { value:"UKI" }) MERGE (node)-[:usedBy { label:"London"}]->(parent);
+MERGE (node:`_code`:`_code_geography` { value:"UKJ" });
+MATCH (parent:`_code_list`:`_code_list_nuts-geography`),(node:`_code`:`_code_geography` { value:"UKJ" }) MERGE (node)-[:usedBy { label:"South East (England)"}]->(parent);
+MERGE (node:`_code`:`_code_geography` { value:"UKK" });
+MATCH (parent:`_code_list`:`_code_list_nuts-geography`),(node:`_code`:`_code_geography` { value:"UKK" }) MERGE (node)-[:usedBy { label:"South West (England)"}]->(parent);
+MERGE (node:`_code`:`_code_geography` { value:"UKL" });
+MATCH (parent:`_code_list`:`_code_list_nuts-geography`),(node:`_code`:`_code_geography` { value:"UKL" }) MERGE (node)-[:usedBy { label:"Wales"}]->(parent);
+MERGE (node:`_code`:`_code_geography` { value:"UKM" });
+MATCH (parent:`_code_list`:`_code_list_nuts-geography`),(node:`_code`:`_code_geography` { value:"UKM" }) MERGE (node)-[:usedBy { label:"Scotland"}]->(parent);
+MERGE (node:`_code`:`_code_geography` { value:"UKN" });
+MATCH (parent:`_code_list`:`_code_list_nuts-geography`),(node:`_code`:`_code_geography` { value:"UKN" }) MERGE (node)-[:usedBy { label:"Northern Ireland"}]->(parent);
+MERGE (node:`_code`:`_code_geography` { value:"T" });
+MATCH (parent:`_code_list`:`_code_list_nuts-geography`),(node:`_code`:`_code_geography` { value:"T" }) MERGE (node)-[:usedBy { label:"Total"}]->(parent);
+MERGE (node:`_code`:`_code_geography` { value:"UKZ" });
+MATCH (parent:`_code_list`:`_code_list_nuts-geography`),(node:`_code`:`_code_geography` { value:"UKZ" }) MERGE (node)-[:usedBy { label:"Extra-regio"}]->(parent);
```

Comment:
`n` missing

Reply:
I also thought this originally but "extra-regio" is the correct name